src/Umbraco.Cms.Integrations.Crm.Dynamics/Configuration/OAuthSettings.cs

```diff
@@ -0,0 +1,20 @@
+﻿
+namespace Umbraco.Cms.Integrations.Crm.Dynamics.Configuration
```

Comment:
Is this included in error?  Looks like it's related to Dynamics and presumably the other work planned to allow customers to bring their own app for authentication.  If so, let's keep that as a separate update.

Reply:
Removed